third_party/cereal/BUILD

```diff
@@ -116,9 +116,7 @@ genrule(
 
 cc_library(
     name = "cereal-cxx",
-    srcs = [
-        "empty.cc",
```

Comment:
IIRC, the empty file is needed for compiling correctly

Reply:
"bazel build --config=darwin heron/..." seems to be working ok for me locally. Doing another clean rebuild now.

Reply:
Builds ok.

INFO: Elapsed time: 1086.264s, Critical Path: 195.24s
INFO: 2515 processes: 2202 local, 313 worker.
INFO: Build completed successfully, 4264 total actions